mu4/notation/internal/notationinteraction.cpp

```diff
@@ -388,11 +388,16 @@ Element* NotationInteraction::hitElement(const QPointF& pos, float width) const
 {
     QList<Ms::Element*> ll = hitElements(pos, width);
     if (ll.isEmpty()) {
-        return nullptr;
+        return hitMeasure(pos).measure;
```

Comment:
why it's not a part of "hitElements" method?

Reply:
fixed